docs/guidelines/platforms/android/guideline_operable_android.md

```diff
@@ -206,16 +206,38 @@ ViewCompat.setAccessibilityDelegate(personalData, object : AccessibilityDelegate
 
 ---
 
-### Page Titles
+### Page Titled (WCAG 2.4.2 - Level A)
 
-*This guideline covers point 2.4.2 Page Titled - Level A of the WCAG standard.*
+> This guideline covers point *2.4.2 Page Titled - Level A of the WCAG standard.*
 
 :white_check_mark: **Success criteria**
 
 Each screen should have a clear, descriptive, and, if possible, unique title that describes the purpose of that screen that will be understandable to all users. Also, it is important to make sure that the title is the first element read when the user enters the screen. This could be achieved by following design guidelines or with the help of setting the `android:accessibilityTraversalBefore` attribute.
 
 If the title is defined using a toolbar with custom behavior or another custom view, it is important to ensure that it will be read using accessibility services.
 
+If you're using Compose, set the `traversalIndex` to -1f to ensure that the TalkBack prioritises it during traversal. This negative value ensures it is read before any elements with a default index of 0f. For layouts that use `TopAppBar`, they are usually prioritised first, but the TalkBack reads the navigation icon before the title. This can be fixed by setting the traversalIndex of title composable to -1f.
```

Comment:
What happens if two elements have the same -1f index?

Reply:
@Aksi0m Assistive technologies take the language (layout direction of the application into account). In Left-to-Right languages, the order would go from top to bottom, from left to right.

Reply:
Can we maybe add this info somewhere in the text. I find that as a very useful info.